exporter/collector/integrationtest/testcase.go

```diff
@@ -95,8 +95,8 @@ func (m *MetricsTestCase) LoadOTLPMetricsInput(
 
 	for i := 0; i < metrics.ResourceMetrics().Len(); i++ {
 		rm := metrics.ResourceMetrics().At(i)
-		for i := 0; i < rm.InstrumentationLibraryMetrics().Len(); i++ {
-			rmi := rm.InstrumentationLibraryMetrics().At(i)
+		for i := 0; i < rm.ScopeMetrics().Len(); i++ {
+			rmi := rm.ScopeMetrics().At(i)
```

Comment:
nit it looks like this has gone through a few renames, maybe `sms` for ScopeMetrics would be more appropriate?

Reply:
done